src/h5web/vis-packs/core/heatmap/HeatmapMesh.tsx

```diff
@@ -114,6 +115,27 @@ const SCALE_SHADER: Record<ScaleType, ScaleShader> = {
       return value >= 0.;
     }`,
   },
+  [ScaleType.Gamma]: {
+    uniforms: (domain: Domain, exponent = 1) => ({
+      min: { value: domain[0] },
+      oneOverRange: { value: 1 / (domain[1] - domain[0]) },
+      gammaExponent: {
+        value: exponent,
+      },
+    }),
+    fragment: `
+    uniform float min;
+    uniform float oneOverRange;
+    uniform float gammaExponent;
+
+    float scale(float value) {
+      return pow(oneOverRange * (value - min), gammaExponent);
+    }
+
+    bool isSupported(float value) {
+      return true;
+    }`,
+  },
```

Comment:
Good stuff with the uniforms defined in the same object 💯 

Maybe just fix the indentation of the fragment string? The extra spaces don't matter anyway, so might as well make it more readable. Also, the `exponent` uniform can fit on a single line.